src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1,23 +1,23 @@
 (ns puppetlabs.puppetdb.scf.storage
   "Catalog persistence
 
-   Catalogs are persisted in a relational database. Roughly speaking,
-   the schema looks like this:
+  Catalogs are persisted in a relational database. Roughly speaking,
```

Comment:
I think all of this whitespace change might be coming in via merges from stable. We auto formatted master but not stable.